elasticsearch_dsl/search.py

```diff
@@ -294,7 +294,6 @@ def _clone(self):
         s = self.__class__(
             using=self._using, index=self._index, doc_type=self._doc_type
         )
-        s._collapse = self._collapse.copy()
```

Comment:
Great catch, thanks! Can you please move `self._collapse = {}` to `Search.__init__` too?

Reply:
the `Search` class already initializes `self._collapse`. The one in `Request` should not be there, I'm removing it.